macro/Cargo.toml

```diff
@@ -27,3 +27,6 @@ syn = { workspace = true }
 quote = { workspace = true }
 subxt-codegen = { workspace = true, features = ["fetch-metadata"] }
 scale-typegen = { workspace = true }
+
+[lints]
+workspace = true
```

Comment:
Do you mind adding this to all crates in subxt?

Reply:
added to all of them, except for examples and testing